smriprep/workflows/outputs.py

```diff
@@ -440,3 +516,20 @@ def _empty_report(in_file=None):
                 <h4 class="elem-title">A previously computed T1w template was provided.</h4>
 """)
     return str(out_file)
+
+
+def _is_native(value):
+    return value == 'native'
+
+
+def _no_native(value):
+    try:
+        return int(value)
+    except Exception:
+        return 1
+
+
+def _drop_path(in_path):
```

Comment:
I haven't really looked into this - it may not be working (although I would expect an error to be raised if the relative path can't be computed)